pandas/core/indexes/base.py

```diff
@@ -5557,6 +5557,25 @@ def _should_fallback_to_positional(self) -> bool:
         missing : np.ndarray[np.intp]
             An indexer into the target of the values not found.
             These correspond to the -1 in the indexer array.
+
+        Examples
+        --------
+        >>> index = pd.Index(['c', 'b', 'a', 'b', 'b'])
+        >>> index.get_indexer_non_unique(['f', 'b', 's'])
+        (array([-1,  1,  3,  4, -1]), array([0, 2]))
+
+        >>> index = pd.Index(['c', 'b', 'a', 'b', 'b'])
+        >>> index.get_indexer_non_unique(['b', 'b'])
+        (array([1, 3, 4, 1, 3, 4]), array([], dtype=int64))
+
+        >>> index = pd.Index(['c', 'b', 'a', 'b', 'b'])
+        >>> index.get_indexer_non_unique(['q', 'r', 't'])
+        (array([-1, -1, -1]), array([0, 1, 2]))
+
+        Notice that the return value is a tuple contains two items.
+        The first item is an array of locations in ``index`` and ``x``
+        is marked by -1, as it is not in ``index``. The second item
+        is a mask for new index given the current index.
```

Comment:
not sure we need to repeat the `return` description here - perhaps, describe the mask given the example? e.g. given that the first and third elements are missing, the mask shows `[0, 2]`

Reply:
agreed, there is no need to repeat `return` sentence. I will add an explanation of what the mask shows.